zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to **Apps** in the bottom-right corner.
+
+1. Then, Go to **Admin Panel** -> **Webhooks**.
+
+1. Click on the **Add** button next to **Outgoing Webhook**.
+
+1. Enter the name of the channel for which you want notifications,
+ in **Send messages from a channel** field.
```

Comment:
Replace with `Set **Send messages from a channel** to the channel you'd like to be notified about.`